packages/core/src/utils/Util.ts

```diff
@@ -835,83 +843,36 @@ export const substanceLiteralToValue = (
   }
 };
 
-const resolveRhsPath = (
-  p: WithContext<Path<A>>,
-): ResolvedPath<A> | FloatV<number> | StrV => {
-  const { name, members, indices } = p.expr; // drop `indices`
-
-  const { subst } = p.context;
-
-  // special handling so that if a Style variable maps to a Substance literal,
-  // then accessing it just gives the value.
-  if (members.length === 0 && indices.length === 0) {
-    if (subst.tag === "StySubSubst" && name.contents.value in subst.contents) {
-      const subObj = subst.contents[name.contents.value];
-      if (subObj.tag === "SubstanceLiteral") {
-        return substanceLiteralToValue(subObj);
-      }
-    }
-
-    if (
-      subst.tag === "CollectionSubst" &&
-      name.contents.value in subst.groupby
-    ) {
-      const subObj = subst.groupby[name.contents.value];
-      if (subObj.tag === "SubstanceLiteral") {
-        return substanceLiteralToValue(subObj);
-      }
-    }
-  }
-
-  return { ...resolveRhsName(p.context, name), members };
-};
-
-const blockPrefix = ({ tag, contents }: LocalVarSubst): string => {
-  switch (tag) {
-    case "LocalVarId": {
-      const [i, j] = contents;
-      return `${i}:${j}:`;
-    }
-    case "NamespaceId": {
-      // locals in a global block point to globals
-      return `${contents}.`;
+export const prettyResolvedStylePath = (p: StylePath<A>): string => {
+  switch (p.tag) {
+    case "Empty":
+      return "";
+    case "Substance": {
+      const o = p.substanceObject;
+      const s = subObjectToUniqueName(o);
+      return `\`${s}\``;
     }
-  }
-};
-
-const prettyPrintResolvedName = ({
-  tag,
-  block,
-  name,
-}: ResolvedName): string => {
-  switch (tag) {
-    case "Global": {
-      return name;
+    case "Namespace":
+      return p.name;
+    case "Unnamed": {
+      const { blockId, substId } = p;
+      return `${blockId}:${substId}`;
```

Comment:
Looks like now local variables are pretty-printed differently than the previous version via `blockPrefix`. I'd prefer fewer noises in the registry diff. 
```suggestion
      return `${blockId}:${substId}:`;
```

Reply:
I did briefly touch on this in the Misc section of the PR. I did this for consistency sake, but can revert it back if necessary.